web_scraper/README.md

```diff
@@ -0,0 +1,23 @@
+# Webscraper
```

Comment:
Do we really need it inhere? Maybe we could move all that code into dlc package? To keep only directly related code in the repo?

Reply:
I think it would actually be a good idea to have this be a part of the GitHub code, since it doesn't need to use any private credentials, and would be easier to debug.